content/influxdb/v2.4/admin/internals/tsi/inspect.md

```diff
@@ -0,0 +1,251 @@
+```
+
+{{< expand-wrapper >}}
+{{% expand "View example output" %}}
+```
+[LOG FILE] L0-00000006.tsl
+Series:		0
+Measurements:	0
+Tag Keys:	0
+Tag Values:	0
+
+[INDEX FILE] L3-00000008.tsi
+Measurements:		3
+  Series data size:	0 (0.0b)
+  Bytes per series:	0.0b
+Tag Keys:		15
+Tag Values:		1025
+  Series:		1700
+  Series data size:	0 (0.0b)
+  Bytes per series:	0.0b
+
+[LOG FILE] L0-00000010.tsl
+Series:		0
+Measurements:	0
+Tag Keys:	0
+Tag Values:	0
+
+[INDEX FILE] L2-00000011.tsi
+Measurements:		1
+  Series data size:	0 (0.0b)
+  Bytes per series:	0.0b
+Tag Keys:		5
+Tag Values:		9
+  Series:		10
+  Series data size:	0 (0.0b)
+  Bytes per series:	0.0b
+```
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+### Output raw series data stored in the index
+
+To output raw series data stored in index files, include the `--series` flag with
+the `influxd inspect dump-tsi` command:
+
+```sh
+influxd inspect dump-tsi \
+  --series \
+  --series-file ~/.influxdbv2/engine/data/056d83f962a08461/_series \
+  ~/.influxdbv2/engine/data/056d83f962a08461/autogen/1023/index
+```
+
+{{< expand-wrapper >}}
+{{% expand "View example output" %}}
+```
+earthquake,code=6000iuad,id=us6000iuad,magType=mww,net=us,title=M\ 5.2\ -\ 101\ km\ SE\ of\ Palca\,\ Peru
+earthquake,code=71377273,id=pr71377273,magType=md,net=pr,title=M\ 1.9\ -\ Puerto\ Rico\ region
+earthquake,code=73794611,id=nc73794611,magType=md,net=nc,title=M\ 0.6\ -\ 13km\ ESE\ of\ Mammoth\ Lakes\,\ CA
+earthquake,code=40361800,id=ci40361800,magType=ml,net=ci,title=M\ 1.3\ -\ 12km\ SE\ of\ Olancha\,\ CA
+earthquake,code=6000itfk,id=us6000itfk,magType=mb,net=us,title=M\ 4.4\ -\ Mindanao\,\ Philippines
+earthquake,code=2022ucrr,id=ok2022ucrr,magType=ml,net=ok,title=M\ 1.4\ -\ 4\ km\ SSE\ of\ Dover\,\ Oklahoma
+earthquake,code=73792706,id=nc73792706,magType=md,net=nc,title=M\ 0.6\ -\ 7km\ W\ of\ Cobb\,\ CA
+earthquake,code=6000isjn,id=us6000isjn,magType=mww,net=us,title=M\ 5.5\ -\ 69\ km\ E\ of\ Hualien\ City\,\ Taiwan
+earthquake,code=022d8mp4dd,id=ak022d8mp4dd,magType=ml,net=ak,title=M\ 1.3\ -\ Southern\ Alaska
+earthquake,code=022dbrb8vb,id=ak022dbrb8vb,magType=ml,net=ak,title=M\ 1.6\ -\ 37\ km\ NE\ of\ Paxson\,\ Alaska
+earthquake,code=6000iu2e,id=us6000iu2e,magType=mb,net=us,title=M\ 4.1\ -\ 81\ km\ WSW\ of\ San\ Antonio\ de\ los\ Cobres\,\ Argentina
+```
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+### Output measurement data stored in the index
+
+To output measurement information stored in index files, include the `--measurement`
+flag with the `influxd inspect dump-tsi` command:
+
+```sh
+influxd inspect dump-tsi \
+  --measurements \
+  --series-file ~/.influxdbv2/engine/data/056d83f962a08461/_series \
+  ~/.influxdbv2/engine/data/056d83f962a08461/autogen/1023/index
+```
+
+{{< expand-wrapper >}}
+{{% expand "View example output" %}}
+```
+Measurement
+earthquake
+explosion
+quarry blast
+
+
+Measurement
+earthquake
+explosion
+ice quake
+quarry blast
+
+
+Measurement
+earthquake
+explosion
```

Comment:
is this output correct? curious why measurement is listed thrice--3 different tsi indexes that include these measurements?

Reply:
Yeah, I just copied this output straight from my terminal. I don't know exactly why it's structured the way it is.